app/src/main/res/layout/view_contact_query.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
not a big deal, but can we rename the file to `view_search.xml` to respect the name `SearchView` ?